Documentation/ProjectHistory.md

```diff
@@ -0,0 +1,8 @@
+# WheelMUD Project History
+
+## Origin
+The WheelMUD initiative originated in 1998. Over the years, the code-base has changed wildly. A couple of times, the project code has been largely or entirely scrapped; we're willing to discuss and recognize when "it's just not working," and we've learned a ton over the years. The earliest attempts were actually Visual Basic initiatives. We're finally at a point where it really feels like our direction is solid and capable of meeting our goals.
+
+## Leadership
+The leadership team here has also, slowly, changed over the years. In February 2015, Fastalanasa has decided to transfer the Champion role over to David Rieman. So, a little about David:
```

Comment:
Hehe, I'm not going to write your bio for you.  I think it would be fine to have a section for bios for devs we've trusted and admitted into this WheelMUD github organization.